src/pages/dashboard/dashboard.component.js

```diff
@@ -3,6 +3,7 @@ import { useSelector } from 'react-redux';
 import Template from '../../components/template/template.component';
 
 const Dashboard = () => {
+  // eslint-disable-next-line
```

Comment:
same here? Why do we need to disable this?

Reply:
Because if we don't disable eslint it will point const user from use selector as initialized but never used, and the project will be compiled with warnings